pkg/fleet/installer/packages/embedded/templates/gen/debrpm/datadog-agent-ddot.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Datadog Distribution of OpenTelemetry Collector
+After=datadog-agent.service datadog-agent-exp.service
```

Comment:
Similar, comment, does it make sense to have `datadog-agent-exp.service` here if it conflicts with it?